src/02-data-types.js

```diff
@@ -79,7 +91,14 @@ function convertToWhisperText(text) {}
  *  checkIfCharacterIsInString("hello there", "a");
  *  //> false
  */
-function checkIfCharacterIsInString(text, character) {}
+function checkIfCharacterIsInString(text, character) {
+  if (text.includes(character)){
```

Comment:
great looking code here Jede.  I'd challenge you to attempt this without ` .includes() `